### DISCUSSION TITLE
XAML preview can't find PathIcon from ResourceDictionary

### DISCUSSION THREAD

**Discussion by jonjonsson** [AvaloniaUI/Avalonia#14347] [General]
I have this PathIcon in my XAML

`<PathIcon Data="{StaticResource clone.img}" />`

At the top of the usercontrol I have
```
<UserControl.Resources>
    <ResourceDictionary>
        <ResourceDictionary.MergedDictionaries>
            <ResourceInclude Source="Resources/Images.axaml"/>
        </ResourceDictionary.MergedDictionaries>
    </ResourceDictionary>
</UserControl.Resources>
```

In Resources/Images.axaml I have
```
<StreamGeometry x:Key="clone.img">
    M6 6h16v4h2V6a2 2 0 0 0-2-2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h4v-2H6Z
    M30 12H14a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2V14a2 2 0 0 0-2-2Zm0 18H14V14h16Z
    M21 28h2v-5h5v-2h-5v-5h-2v5h-5v2h5v5Z
</StreamGeometry>
```

This compiles fine and works. Only problem is the XAML preview doesn't work anymore:

> Static resource 'clone.img' not found.	

Do I need to specify somewhere that the preview should use this file?

**Comment by maxkatz6** [comment]
It's hard to tell without a specific minimal repro, where we can see which files and how are referenced. 
This ResourceInclude should be either in the same file as StaticResource, or in the Application.

**Comment by jonjonsson** [comment]
I got around this by putting my Images.axaml into /Designer/Resources as well. This is kind of pita solution though, need to maintain 2 instances of the same file.

**Comment by workgroupengineering** [comment]
Strange, your problem sound like https://github.com/AvaloniaUI/AvaloniaVS/issues/324. What is Avallonia's version?

**Reply by jonjonsson** [reply]
I'm on 11.0.4.  I will do an update and report back!

**Comment by jonjonsson** [comment]
Thanks for the help guys.

Turns out that using 

`<ResourceInclude` Source="Resources/Images.axaml"/>`

works when compiling but the design preview needs to have the full path  "/Views/Resources/Images.axaml"

Cheers!

**Reply by workgroupengineering** [reply]
Try with
`<ResourceInclude Source="./Resources/Images.axaml"/>`
